### PR TITLE
new start turtle pull request

### PR DESCRIPTION
**Pull request by redashu**

### CODE REVIEW

**Review by lnbspace — COMMENTED** [review]
Need to apply comments

**Comment by redashu** [comment]
done with comments

File: star.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Python program to draw star
+# using Turtle Programming
+import turtle
+
+star = turtle.Turtle()
+
+for i in range(50):
```
Comment (lnbspace) [inline star.py:7]: # this is is not good

Comment (redashu) [reply star.py:7]: ok

**Pull request merged**